integr8ly/integreatly-operator
# Added RHOAM specific SLO dashboard

## Description

[MGDAPI-309](https://issues.redhat.com/browse/MGDAPI-309)

Created a new SLO dashboard specific to RHOAM installs.

## Verification Steps
We need to verify that both the new and existing dashboard SLO summary dashboards are created as per the install type

### Verify managed install type
- To ensure the correct dashboard get created for managed installs, complete an install of type `managed`
- Ensure that the install completes, and the `critical-slo-summary` Grafana dashboard is created and populated with the `SLO Summary` and `redhat-rhmi*` panels. This install is to test that there are no changes to this dashboard.

### Verify managed-api install type
- Complete an install of type `managed-api`:
 ```
 export NAMESPACE_PREFIX=redhat-managed-api- && export INSTALLATION_TYPE=managed-api
 make cluster/prepare/local
 make code/run
```
- Ensure that the install completes, and the `critical-slo-summary` Grafana dashboard is created and populated only with the `SLO Summary`, `3scale`, `rhsso` and `user-rhsso` panels, and the `namespaceCustom` and `product` drop downs contain the correct namspaces/products only (i.e, no RHMI specific products should be present)
- Complete the following manual test, ensuring it passes: [MGDAPI-269](https://issues.redhat.com/browse/MGDAPI-269)

## Type of change




## Checklist
